kuma/core/middleware.py

```diff
@@ -126,16 +134,14 @@ class LocaleMiddleware(MiddlewareMixin):
     * Don't include "Vary: Accept-Language" header
     * Add caching headers to locale redirects
     """
-    response_redirect_class = HttpResponseRedirect
 
-    def process_request(self, request):
-        """Activate the language, based on the request."""
-        language = get_language_from_request(request)
-        translation.activate(language)
-        request.LANGUAGE_CODE = language
+    def __call__(self, request):
+        # Activate the language, based on the request.
```

Comment:
You've made this comment redundant with the function name 🙂. However, it's not 100% clear from the name that ``request`` gains a ``LANGUAGE_CODE`` parameter, maybe that's worth including in the comment.